manual/installation.tex

```diff
@@ -1562,26 +1555,25 @@ \subsubsection{NiO performance tests}
 performance-NiO-cpu-a1024-e12288& S256 & 1024 & 12288 & 6144 \\
 \hline
 \end{tabular}
-  \caption{System sizes and names for NiO performance tests. GPU performance
-    tests are named similarly but have different walker counts.}
+
   \label{tab:niotests}
 \end{center}
 \end{table}
 
 \subsection{Troubleshooting tests}
-ctest reports briefly pass or fail of tests in printout and also collects all the standard outputs to help investigating how tests fail.
+Ctest reports briefly pass or fail of tests in printout and also collects all the standard outputs to help investigating how tests fail.
```

Comment:
I think CTest should be capitalized in the same style as CMake

Reply:
Fixed throughout